ci-operator/step-registry/gather/gather-chain.yaml

```diff
@@ -4,5 +4,6 @@ chain:
   - ref: gather-must-gather
   - ref: gather-extra
   - ref: gather-audit-logs
+  - ref: gather-log-to-loki
```

Comment:
No, this is a bad idea. This would dump all data immediately instead of continuously sending, so timestamps will be off. Also, we'll get no data if the pod cannot be reached - and the script doesn't wait for it to finish sending

You need to add a new container which dumps prometheus adapter logs to stdout (see https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_audit_logs#L53-L70) and add new promtail stage which converts this in "audit: prometheus-adapter" logs similar to https://github.com/openshift/release/pull/25344.

Reply:
Thanks Vadim :) I'm going to try out this method.